casper/src/test/scala/coop/rchain/casper/util/GenesisBuilder.scala

```diff
@@ -61,6 +61,37 @@ object GenesisBuilder {
     "047f0f0f5bbe1d6d1a8dac4d88a3957851940f39a57cd89d55fe25b536ab67e6d76fd3f365c83e5bfe11fe7117e549b1ae3dd39bfc867d1c725a4177692c4e7754"
   )
 
+  def buildGenesisParameters(
+      validatorKeyPairs: Iterable[(PrivateKey, PublicKey)],
+      genesisVaults: Seq[(PrivateKey, Long)],
+      bonds: Map[PublicKey, Long]
+  ): GenesisParameters =
+    (
+      validatorKeyPairs,
+      genesisVaults.map { case (k, _) => (k, Secp256k1.toPublic(k)) },
+      Genesis(
+        shardId = "root",
+        timestamp = 0L,
+        proofOfStake = ProofOfStake(
+          minimumBond = 1L,
+          maximumBond = Long.MaxValue,
+          // Epoch length is set to large number to prevent trigger of epoch change
+          // in PoS close block method, which causes block merge conflicts
+          // - epoch change can be set as a parameter in Rholang tests (e.g. PoSSpec)
+          epochLength = 1000,
+          quarantineLength = 50000,
+          numberOfActiveValidators = 100,
+          validators = bonds.map(Validator.tupled).toSeq,
+          posMultiSigPublicKeys = defaultPosMultiSigPublicKeys,
+          posMultiSigQuorum = defaultPosMultiSigPublicKeys.length - 1
+        ),
+        vaults = genesisVaults.toList.map {
+          case (p, s) => Vault(RevAddress.fromPublicKey(Secp256k1.toPublic(p)).get, s)
+        },
+        supply = Long.MaxValue,
+        blockNumber = 0
+      )
+    )
```

Comment:
I was trying to refactor codes in this file but end up with a lot of unrelated changes and failing tests. I would just keep it here and see if casper change need to consider this.